usersprompt.js

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Good work on this but I noticed that your code isn't properly formatted. If you are using vscode on windows, you can format your code by pressing `Alt + Shift + f`, If you are on Linux then `Ctrl + Shift + I`. If you are using another editor or IDE, you can look for how to format its code using any search engine